readme.md

```diff
@@ -153,6 +153,9 @@ SMTP_USERNAME = '<smtp_user>'
 SMTP_PASSWORD = '<smtp_pass>'
 
 LOG_FILE = '/var/log/django.log'
+
+# Configuration of the number of threads to be used for predictions - Set to CPU cores + 1 if dedicated machine
+NUM_THREADS = 5
```

Comment:
Can we add this to the main `settings.py` with a default value like 2 in order to avoid breaking functionality for those that won't update their `local_settings`.
Another option is to cleverly catch the broken import of the settings and notify the user he needs to update the `local_settings`, but I'm kinda leaning on the first option.